### ISSUE TITLE
If I claim my content, its origin on SE is no longer relevant: can we stop showing it on the post?

### ISSUE THREAD

**Issue by cellio** [ArtOfCode-/qpixel#38]
I claimed my content, which I interpret to mean "I am that person, and I am now granting a license for QPixel to use it" (same as if I created a new post).  When this happens, could we remove the "imported from SE with license (whatever)" notice from the post?  It's fine to keep that in the history (and we'd want to for the timing and score), but I don't think we should keep emphasizing it in the primary UI.  Over time we want this site to look more like the current community and less like a clone, and even for older content, I think new ownership claims should count as "current community".

I guess that means claiming the content should properly add another history entry.  I don't think very many of us have claimed content so far, so up to you whether we just shrug in those cases.

**Comment by ArtOfCode-** [comment]
Working on this, but there's quite a bit more to it than on the surface. Removing the attribution notice means making sure the user agrees to relicense the content to the license that the site is using, which is... what exactly? That's going to need a configurable site setting, which in turn means I need to write a basic ToS document saying "by having an account here you agree to license content under (X license)".

**Comment by ArtOfCode-** [comment]
So that I'm not half-assing this feature and having to come back to it later, this is currently blocked by https://github.com/ArtOfCode-/qpixel/projects/2#card-30292923.

**Comment by cellio** [comment]
Oh!  Yes, that's a good point -- we don't have TOS yet to address the license.  Good catch.

**Comment by ArtOfCode-** [comment]
b4bf64f96bdb635a562074aeb440584b82040064 is the majority of this work done, but it still needs heavy testing and polishing before I'd call it prod-ready.
